### DISCUSSION TITLE
Browse Media List

### DISCUSSION THREAD

**Discussion by Schmu1605** [punxaphil/custom-sonos-card#68] [General]
is it possible to display the media browser as a list instead of in grid mode?

**Comment by punxaphil** [comment]
Sure, I'll add it to the todo-list for future updates

**Comment by BrianKarlsen** [comment]
This would be great! :)

**Comment by punxaphil** [comment]
@BrianKarlsen @Schmu1605 Please try out this beta: https://github.com/johanfrick/custom-sonos-card/releases/tag/v5.14.0-beta_media-browser-list
Does it work for you?

**Reply by BrianKarlsen** [reply]
When installing your beta I just overwrite the custom-sonos-vard.js file right?

**Reply by punxaphil** [reply]
I recommend using HACS

**Reply by BrianKarlsen** [reply]
Still grid view
(forgot about the beta feature un HACS... )

**Reply by punxaphil** [reply]
Forgot to tell you, you need to configure it:
```
mediaBrowserItemsAsList: true
```

**Reply by BrianKarlsen** [reply]
is this correct?
        cards:
          - type: custom:custom-sonos-card
            singleSectionMode: media browser
            mediaBrowserItemsAsList: true
            layout:
              mediaItem:
                width: 25%

**Reply by BrianKarlsen** [reply]
indents wrong in this post :)

**Reply by punxaphil** [reply]
That should work, but the mediaItem layout makes no diff. Since it's list it will always be 100%.
It could also be simplified as:
```
type: custom:sonos-media-browser
mediaBrowserItemsAsList: true
```
Please double check that you have this version of the card: `v5.14.0-beta_media-browser-list`

**Reply by BrianKarlsen** [reply]
I'ts Working!!
how do I change layout to make the media list smaller. Using landscape mode on a tablet so 50/50 dosent look good.
![image](https://user-images.githubusercontent.com/24981461/204370550-a13b656d-bf18-4c05-b01c-4bfbda17b421.png)

**Reply by BrianKarlsen** [reply]
![image](https://user-images.githubusercontent.com/24981461/204370979-a4425086-5901-4f61-9e9e-eeec620f15e9.png)

**Reply by punxaphil** [reply]
Great it's working. Lot's of examples of layouts in the README, please have a look there.
https://github.com/johanfrick/custom-sonos-card#readme
